diff --git a/debian/rules b/debian/rules
index 15e0592..c06324d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -200,6 +200,7 @@ override_dh_binary-modules: override_dh_prep-deb-files override_dh_configure_mod
 	dh_prep
 
 	$(MAKE) -C $(CURDIR)/module modules
+	find $(CURDIR/module) -name "*.ko" -exec $(SIGNFILE) $(HASH) $(KEY) $(CERT) {} \;
 
 	dh_install -p${pmodules}
 	dh_installdocs -p${pmodules}
